app/Http/Controllers/UserController.php

```diff
@@ -125,4 +125,339 @@ public function destroy(User $user)
             return redirect()->back()->with('messageFail', 'Delete failed. Something went wrong');
         }
     }
+
+    public function testUpdateUserSuccessFeature()
+    {
+        $admin = factory(User::class)->create(['role' => UserType::ADMIN]);
+        $this->actingAs($admin);
+        $user = factory(User::class)->create(
+            [
+                'name' => "Name Create",
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Strings.DoubleQuoteUsage.NotRequired</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 135</br>String "Name Create" does not require double quotes; use single quotes instead